packages/tools/fozzie/src/scss/objects/_buttons.scss

```diff
@@ -108,9 +107,6 @@
         color: $btn-default-color;
         text-decoration: none;
 
-        &:focus {
```

Comment:
Should this be replaced by a new focus style for fozzie buttons? (these are likely used in things like Payments/OrderWeb). Not critical, but may need something added for these in a subsequent PR.

Reply:
We'll check this post-merge, and patch if it does break